homework-2.rb

```diff
@@ -0,0 +1,386 @@
+=begin
+  Assignment #2, Tic-Tac-Toe
+=end
+
+require 'google_drive'
+
+$winner = nil
+def checkForWin(xy)
+  session = GoogleDrive::Session.from_config("homework-2_config.json")
```

Comment:
You should make these globals instead of initializing it each time. Same with line 10 with the spreadsheet creation.